src/types/emr/permission/permission.ts

```diff
@@ -4,3 +4,12 @@ export interface Permission {
   context: string;
   description: string;
 }
+
+export interface Permissions {
+  permissions: string[];
+}
+
+export interface FacilityPermissions extends Permissions {
+  root_org_permissions: string[];
```

Comment:
In order to cleanly manage all permission based rendering, this should ideally be an enum of text values, one for each backend permission.

Making it a follow up issue is also fine